pkg/prometheus/server/operator.go

```diff
@@ -237,6 +239,20 @@ func New(ctx context.Context, conf operator.Config, logger log.Logger, r prometh
 		return nil, errors.Wrap(err, "error creating probe informers")
 	}
 
+	c.sconInfs, err = informers.NewInformersForResource(
+		informers.NewMonitoringInformerFactories(
+			c.config.Namespaces.AllowList,
+			c.config.Namespaces.DenyList,
+			mclient,
+			resyncPeriod,
+			nil,
+		),
+		monitoringv1.SchemeGroupVersion.WithResource(monitoringv1alpha1.ScrapeConfigName),
```

Comment:
Tried the e2e test locally and saw this log

```
instantiating prometheus controller failed: error creating scrapeconfig informereos.com/v1, Resource=scrapeconfigs: no informer found for monitoring.coreos.com/
```

It's super easy to miss this one 😅, it should fix the problem
```suggestion
		monitoringv1alpha1.SchemeGroupVersion.WithResource(monitoringv1alpha1.ScrapeConfigName),
```

Reply:
Thanks! Pushed in 2197decab395c9104ad36f6f1f99f5f51e19ecc1